examples_test.go

```diff
@@ -231,33 +274,37 @@ func Example_direct_cdp() {
 	// rod=test
 }
 
-// Shows how to subscribe events.
+// Example_handle_events is an example showing how we can use Rod to subscribe
+// to events.
 func Example_handle_events() {
 	browser := rod.New().Timeout(time.Minute).Connect()
 	defer browser.Close()
 
 	go browser.EachEvent(func(e *proto.TargetTargetCreated) {
-		// if it's not a page return
+		// Only listen when this event is called. Otherwise we can exit.
```

Comment:
This description may confuse users. The event is filtered by Golang's `reflect`, the input type is the filter, `func(e *proto.TargetTargetCreated)` is the filter, other types of events won't come.

The reason for this `if-return` here is because this type of event can be triggered by a new service worker, a new incognito browser, and other types of page.